cypress.json

```diff
@@ -1,3 +1,4 @@
 {
-  "baseUrl": "http://localhost:5000"
+  "baseUrl": "http://localhost:5000",
+  "projectId": "hgum62"
```

Comment:
For my own knowledge, where did this `projectId` come from?

Reply:
When I added this project to our Cypress dashboard it generated a ID and secret key for us.

Combining those two allows test recordings to be shown in our dashboard.